end2end-test-examples/echo-client/src/main/java/io/grpc/echo/EchoClient.java

```diff
@@ -148,24 +150,47 @@ private String generatePayload(int numBytes) {
     return sb.toString();
   }
 
+  void streamingEcho() {
+    long start = 0;
+    try {
+      StreamEchoRequest request = StreamEchoRequest.newBuilder()
+          .setMessageCount(args.numRpcs)
+          .setMessageInterval(Math.max(args.interval, STREAMING_MIN_INTERVAL))
+          .build();
+      start = System.currentTimeMillis();
+      Iterator<EchoResponse> iter = blockingStub.echoStream(request);
+      for (long counter = 0; iter.hasNext(); ++counter) {
+        EchoResponse resp = iter.next();
+        logger.info(String.format("Got %d EchoResponse", counter));
```

Comment:
do we need this info log statement. if doing 100 rpc's, this could produce a bit of log.
(maybe I am missing its usefulness)

Reply:
I think it is ok here, because this is where we receive server-streaming response. Each response comes with a minimum 1-second delay and we want to see how the streaming is going, so I also added the duration.